script/examples/0.0.1/slurm.yml

```diff
@@ -3,7 +3,7 @@ version: 0.0.1
 # Shell would be accepted to indicate a single line shell command (or similar)
 login_node_check:
   type: script
-  shell: "ping login 1"
+  run: "ping login 1"
```

Comment:
Not sure about the YAML syntax i think it should be ok to have ``run: "ping login 1" or should we be consistent
and have
```
  run: |
ping login  1
```

Reply:
Both of those cases are strings, which is what run accepts. The only difference is that the second (with the pipe) is just yaml synax for allowing more than one line.